### DISCUSSION TITLE
Embed build time into humans.txt file

### DISCUSSION THREAD

**Discussion by pke** [vercel/next.js#28591] [Help]
In my humans.txt file I'd like to embed the build time of the site (last update).
Since its a text file essentially and not a tsx or mdx I might have to involve some trickery I guess.

Or could I use a mdx file and `getStaticProps` and somehow rewrite the path of the file to be "humans.txt"?

My idea would be something like that or using a build time environment variable?

```txt
/* SITE */
    Last Update: ${new Date().toISOString()}
```

**Comment by dsernst** [comment]
You could add a script to your git commit hook to automatically update your file then.

https://www.npmjs.com/package/husky for example makes this pretty easy.

**Comment by stigkj** [comment]
Or have something in `package.json` for the `build` task that runs a script.
